.eslintrc.js

```diff
@@ -44,6 +44,11 @@ module.exports = {
     '@typescript-eslint/no-non-null-assertion': 'off',
     '@typescript-eslint/no-this-alias': 'off',
     'func-names': 'error',
+    'import/first': 'error',
+    'import/newline-after-import': 'error',
+    'import/no-duplicates': 'error',
+    'import/order': ['error', {'newlines-between': 'always'}],
+    'no-console': 'error',
```

Comment:
Actually, I use `console.log` in [`src/bidiTab/mapperTabPage.ts`](https://github.com/GoogleChromeLabs/chromium-bidi/blob/1a62b7ea8822291bd88aabe42a3fbf608f232756/src/bidiTab/mapperTabPage.ts#L67). Current `logger` does not log anything on the server side, which makes debugging really hard. Consider adding `debug` flag on the server side, which would send all the logs to the server. Until that `console.log` is the only way to get the logs.

Reply:
This just means we cannot _check in_ code with `console.*`. You can still use it locally.

I will downgrade it to `warn` instead, which is a sweet spot.